Sources/_TestingInternals/Discovery.cpp

```diff
@@ -86,6 +86,23 @@ struct SWTRelativePointer {
   }
 };
 
+/// A type representing a 32-bit absolute function pointer, usually used on platforms
+/// where relative function pointers are not supported.
+///
+/// This type is derived from `AbsoluteFunctionPointer` in the Swift repository.
+template <typename T, int32_t maskValue = 0>
+class SWTAbsoluteFunctionPointer {
+  T *Pointer;
+  static_assert(sizeof(T *) == sizeof(int32_t),
```

Comment:
This `static_assert()` can just be on one line, I think. Code doesn't need to wrap at 80 columns, just comments.